assets/js/googlesitekit/data/create-error-store.test.js

```diff
@@ -170,47 +170,101 @@ describe( 'createErrorStore store', () => {
 	} );
 
 	describe( 'selectors', () => {
-		describe.each( [ 'getErrorForSelector', 'getErrorForAction' ] )(
-			'%s',
-			( selectorName ) => {
-				const baseNameParam =
-					selectorName === 'getErrorForSelector'
-						? 'selectorName'
-						: 'actionName';
-
-				it( `requires a \`${ baseNameParam }\` param`, () => {
-					expect( () => {
-						select[ selectorName ]();
-					} ).toThrow( `${ baseNameParam } is required.` );
-				} );
+		describe( 'getErrorForSelector', () => {
+			const baseNameParam = 'selectorName';
```

Comment:
This constant isn't needed anymore because now `getErrorForSelector` and `getErrorForAction` selectors have separate tests. Please, rename all test names not to use this variable and remove it.